kazoo/recipe/lock.py

```diff
@@ -555,6 +566,10 @@ def _inner_release(self):
         self.is_acquired = False
         return True
 
+    def break_lock(self):
```

Comment:
I am flying a bit blind here. This works for the case I have for tooz. Beyond that, I'm entirely open to suggestions.